apps/dashboard/src/lib/components/Org/Settings/OrgSettings.svelte

```diff
@@ -12,8 +12,19 @@
   import SectionTitle from '../SectionTitle.svelte';
   import { t } from '$lib/utils/functions/translations';
   import { isFreePlan } from '$lib/utils/store/org';
+  import { updateOrgNameValidation } from '$lib/utils/functions/validator';
 
   let avatar;
+  let orgName = '';
```

Comment:
@codiini is storing this variable actually necessary?

Can't we just use `$currentOrg.name`?

Reply:
@rotimi-best You're right. I've updated the PR.